integration/spark-common/src/main/scala/org/apache/carbondata/spark/KeyVal.scala

```diff
@@ -141,9 +141,10 @@ class RestructureResultImpl extends RestructureResult[Int, Boolean] {
 }
 
 trait RefreshResult[K, V] extends Serializable {
-  def getKey(key: String, value: Boolean): (K, V)
+  def getKey(key: String, value: (String, Boolean)): (K, V)
```

Comment:
please add comment to describe the modification

Reply:
comments added to describe the structure